tensorflow/core/kernels/mkl_transpose_op.cc

```diff
@@ -158,7 +153,7 @@ Status MKLTransposeND(OpKernelContext* context, const Tensor& in_tensor,
 #else
     net.push_back(FindOrCreateReorder<T>(in.GetUsrMem(), out.GetUsrMem()));
     transpose_stream->submit(net).wait();
-#endif
+#endif //ENABLE_MKLDNN_V1
```

Comment:
Nit: two spaces after `#endif` and one space after `//`.
```suggestion
#endif  // ENABLE_MKLDNN_V1
```

Reply:
How do i just accept the change?

Reply:
You can just click the `commit suggestion` button, I think. I haven't seen the change reflected on my side, btw.

Reply:
Oh I see it now.